dockstore-webservice/src/main/java/io/dockstore/webservice/resources/HostedWorkflowResource.java

```diff
@@ -177,10 +183,15 @@ public Workflow editHosted(User user, Long entryId, Set<SourceFile> sourceFiles)
     @Path("/hostedEntry/{entryId}")
     @Timed
     @UnitOfWork
-    @ApiOperation(nickname = "addZip", value = "Creates a new revision of a hosted workflow from a zip",
-            authorizations = {@Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Workflow.class)
-    public Workflow addZip(@ApiParam(hidden = true) @Auth User user, @ApiParam(value = "hosted entry ID")
-        @PathParam("entryId") Long entryId,  @FormDataParam("file") InputStream payload) {
+    @ApiOperation(nickname = "addZip", value = "Creates a new revision of a hosted workflow from a zip", authorizations = {
+        @Authorization(value = JWT_SECURITY_DEFINITION_NAME)}, response = Workflow.class)
+    @Deprecated(since = "1.9.0")
+    @Operation(operationId = "addZip", summary = "Creates a new revision of a hosted workflow from a zip", security = @SecurityRequirement(name = "bearer"), deprecated = true)
```

Comment:
Make a private static final thing for "addZip" and "Creates a new revision of a hosted workflow from a zip"